### DISCUSSION TITLE
zzz 11

### DISCUSSION THREAD

**Discussion by flyhig** [share121/inter-knot#1315] [其他]
What if I forget that there are two teams in the abyss?
![Screenshot_2024-08-02-22-24-10-567_com miHoYo Nap](https://github.com/user-attachments/assets/1f801c59-0a10-44f1-a5f8-82b2216d0f00)

**Comment by Kaohaaa** [comment]
Dont worry, one is enough. 
just press alt+F4, all enemies will be kiled

**Reply by 0d00210** [reply]
you are funny

**Comment by gaojiaobei** [comment]
lol
